content/php/concepts/arrays/terms/array-reverse/array-reverse.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'array_reverse()'
+Description: 'Reverses the items in an array and returns the reversed array.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_reverse()`** function reverses the items in an array and returns that reversed array.
+
+## Syntax
+
+```pseudo
+array_reverse(array, preserve_keys)
+```
+
+- `array`: The array to reverse.
+- `preserve_keys`: Optional argument that defaults to `false` if omitted. If set to `true`, the keys of the original, non-reversed array will be preserved.
+> **Note:** If the keys used are non-numeric, they will always be preserved, regardless of whether the `preserve_keys` argument is used.
```

Comment:
add an empty line between the last parameter and the note

Reply:
Done!